public/package.json

```diff
@@ -8,7 +8,7 @@
     "@types/react": "^16.14.2",
     "@types/react-dom": "^16.9.10",
     "google-protobuf": "^3.15.0-rc.1",
```

Comment:
This package seems awfully old and in rc-version... Seems like we should update to [3.21.2](https://www.npmjs.com/package/google-protobuf).

Reply:
I tried updating and running webpack gives:

```sh
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  npm.google-protobuf.6a5af5a33cf71dd652c5.bundle.js (247 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  protobuf (248 KiB)
      runtime.d7bb87a22f8c26f1b120.bundle.js
      npm.google-protobuf.6a5af5a33cf71dd652c5.bundle.js
      protobuf.5c4447d2265d96b49c3f.bundle.js


WARNING in webpack performance recommendations:
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/

webpack 5.73.0 compiled with 3 warnings in 17270 ms
```